tests/DevApps/ciam/myWebApi/Properties/launchSettings.json

```diff
@@ -10,14 +10,14 @@
   },
   "profiles": {
     "https": {
```

Comment:
Should it ever use the `https` profile?

Reply:
Not sure, maybe for testing some specific scenarios. I would leave it there